File: sample/src/androidTest/java/com/emarsys/sample/testutils/SampleAppTestRunner.java
package com.emarsys.sample.testutils;

import android.app.Application;
import android.content.Context;
import androidx.test.runner.AndroidJUnitRunner;

public class SampleAppTestRunner extends AndroidJUnitRunner {
    @Override
    public Application newApplication(ClassLoader cl, String className, Context context) throws InstantiationException, IllegalAccessException, ClassNotFoundException {
        return super.newApplication(cl, InstrumentedApplication.class.getName(), context);
    }
}

File: sample/src/androidTest/java/com/emarsys/sample/testutils/SampleAppTestRunner.kt
package com.emarsys.sample.testutils

import android.app.Application
import android.content.Context

import androidx.test.runner.AndroidJUnitRunner

class SampleAppTestRunner : AndroidJUnitRunner() {
    @Throws(InstantiationException::class, IllegalAccessException::class, ClassNotFoundException::class)
    override fun newApplication(cl: ClassLoader, className: String, context: Context): Application {
        return super.newApplication(cl, Application::class.java.name, context)
    }
}